packages/graphql-language-service-types/src/index.ts

```diff
@@ -277,28 +282,26 @@ export type ObjectTypeInfo = {
   definition: TypeDefinitionNode;
 };
 
+export type WatchmanSubscriptionResult = {
+  root: string;
+  subscription: string;
+  is_fresh_instance?: boolean;
+  files: {
+    name: string;
+    size: number;
+    mtime: number;
+    exists: boolean;
+    type?: 'f' | 'd' | 's';
+  }[];
+};
+
 export type CustomValidationRule = (
   context: ValidationContext,
 ) => Record<string, any>;
 
-export type Diagnostic = {
-  range: Range;
-  severity?: number;
-  code?: number | string;
-  source?: string;
-  message: string;
-};
+export type Diagnostic = DiagnosticType;
 
-export type CompletionItem = {
```

Comment:
hmm... i think i fixed this in terms of how `isDeprecated` matches up with `deprecated` in the built-in vscode interface, but that should be mapped to our built in fields, and I think we should extend `vscode-languageserver-protocol` `CompletionItem` to handle both.